### PR TITLE
Support sending binary buffer over web socket in Android

### PR DESCRIPTION
**Pull request by sreesharp**
This PR is to support the binary buffer send in Websocket.  Similar to the iOS support PR https://github.com/facebook/react-native/pull/6327

### CODE REVIEW

File: ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java [hunk 1]
```diff
@@ -182,6 +182,21 @@ public void send(String message, int id) {
...
+  @ReactMethod
+  public void sendBinary(Buffer buffer, int id) {
+      WebSocket client = mWebSocketConnections.get(id);
+      if (client == null) {
+          // This is a programmer error
+          throw new RuntimeException("Cannot send a message. Unknown WebSocket id " + id);
+      }
+      try {
+          client.sendMessage(
+                  WebSocket.PayloadType.BINARY, buffer);
```
Comment (christopherdro) [inline ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java]: Can we get this on a single line?

**Comment by dmmiller** [comment]
Did you test this?  Can you indicate how?

File: ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java (hunk 1 above)
Comment (sreesharp) [inline ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java]: @christopherdro Thanks. updated.

**Comment by sreesharp** [comment]
@dmmiller  In order to test this feature, first we need to apply the patch from https://github.com/facebook/react-native/pull/6327/files.  Mainly the JS files if we are concerned about android.  I have applied the entire patch, and now it works in both iOS and Android.  There is another test case mentioned here https://github.com/facebook/react-native/pull/6190, and that should also work now in Android with this PR.

**Pull request closed without merging**